src/Sulu/Bundle/SnippetBundle/Admin/WebspaceContentNavigationProvider.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SnippetBundle\Admin;
+
+use Sulu\Bundle\AdminBundle\Navigation\ContentNavigationItem;
+use Sulu\Bundle\AdminBundle\Navigation\ContentNavigationProviderInterface;
+
+class WebspaceContentNavigationProvider implements ContentNavigationProviderInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getNavigationItems(array $options = [])
+    {
+        $snippets = new ContentNavigationItem('content-navigation.webspace.snippets');
+        $snippets->setId('tab-snippets');
+        $snippets->setAction('snippets');
+        $snippets->setPosition(25);
```

Comment:
because i thought that the "main" bundle should set numbers factor 10 and the other bundles can use numbers in between! otherwise contentbundle has to know that snippet bundle uses number 30